### PR TITLE
Gaze con assistive

### PR DESCRIPTION
**Pull request by mikelgg93**

### CODE REVIEW

**Comment by mikelgg93** [comment]
@nadje

- [ ] Upload an image for the main alpha labs page, to showcase this article at media/alpha-lab
- [ ] Changes the description and image path of the card here:

https://github.com/pupil-labs/pupil-docs/blob/cdb3074bae1aa55ee3c1a4a01c2c98740c49d0ec/src/alpha-lab/README.md?plain=1#L131

- [ ] **OPT:** The description of the sidebar is too long, perhaps we would like to use a shorter one at config.js.

File: src/alpha-lab/gaze_contingency_assistive.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: A practical guide to implementing gaze contingency in assistive technology
+description: "Gaze contingent systems for assistive technology"
+permalink: /alpha-lab/gaze-contingency-assistive
+tags: [Neon]
+---
+# A practical guide to implementing gaze contingency in assistive technology
```
Comment (willpatera) [inline src/alpha-lab/gaze_contingency_assistive.md:7]: In assistive technology or
for assistive technology?

File: src/alpha-lab/gaze_contingency_assistive.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+<TagLinks />
+<Youtube src="cuvWqVOAc5M"/>
+
+::: tip
+Imagine a world where transformative assistive solutions allow you to browse the internet with a mere glance or write an email using only your eyes. This is not science fiction; it is the realm of gaze-contingent technology.
```
Comment (willpatera) [inline src/alpha-lab/gaze_contingency_assistive.md:13]: allow or enable?

File: src/alpha-lab/gaze_contingency_assistive.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+coordinates using a [homography](https://en.m.wikipedia.org/wiki/Homography_(computer_vision)) approach like the [Marker Mapper](/enrichments/marker-mapper/) 
+enrichment we offer in Pupil Cloud as a post-hoc solution. The heavy lifting of all this is handled by 
+our [Real-time Screen Gaze](https://github.com/pupil-labs/realtime-screen-gaze/) package (written for this guide).
+
+**Gaze-controlling a mouse**
+
+The second challenge is using screen-mapped gaze to control an input device, e.g., a mouse. To demonstrate how to do this, we 
+wrote the [Gaze-controlled Cursor Demo](https://github.com/pupil-labs/gaze-controlled-cursor-demo). This demo leverages the 
+[Real-time Screen Gaze](https://github.com/pupil-labs/realtime-screen-gaze/) package to obtain gaze in screen-based coordinates, and then uses 
+that to control a mouse in a custom browser window, as shown in the video above. A simple dwell-time filter implemented
```
Comment (willpatera) [inline src/alpha-lab/gaze_contingency_assistive.md:47]: do we need to define dwell time?

Comment (willpatera) [reply src/alpha-lab/gaze_contingency_assistive.md:47]: I don't want to make this longer - so up to the authors to decide if definition is needed.

File: src/alpha-lab/gaze_contingency_assistive.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+The second challenge is using screen-mapped gaze to control an input device, e.g., a mouse. To demonstrate how to do this, we 
+wrote the [Gaze-controlled Cursor Demo](https://github.com/pupil-labs/gaze-controlled-cursor-demo). This demo leverages the 
+[Real-time Screen Gaze](https://github.com/pupil-labs/realtime-screen-gaze/) package to obtain gaze in screen-based coordinates, and then uses 
+that to control a mouse in a custom browser window, as shown in the video above. A simple dwell-time filter implemented 
+in the demo enables mouse clicks when gaze hovers over different elements of the browser.
+
+**It’s your turn…**
+
+Follow the steps in the next section and in no time you will be able to use your gaze to navigate a website and fixate on different parts to trigger a mouse click with your eyes.
```
Comment (willpatera) [inline src/alpha-lab/gaze_contingency_assistive.md:52]: Consider omitting "in no time"

**Review by willpatera — COMMENTED** [review]
Just minor questions/comments, otherwise good to go.

File: src/alpha-lab/gaze_contingency_assistive.md (hunk 1 above)
Comment (N-M-T) [reply src/alpha-lab/gaze_contingency_assistive.md:7]: Should be for

File: src/alpha-lab/gaze_contingency_assistive.md (hunk 3 above)
Comment (N-M-T) [reply src/alpha-lab/gaze_contingency_assistive.md:47]: I think it's pretty self-explanatory. Let's leave it.

File: src/alpha-lab/gaze_contingency_assistive.md (hunk 4 above)
Comment (N-M-T) [reply src/alpha-lab/gaze_contingency_assistive.md:52]: Omitted!

**Pull request merged**